pkg/cmd/indices/config/export/export.go

```diff
@@ -28,19 +29,20 @@ func NewExportCmd(f *cmdutil.Factory) *cobra.Command {
 		Use:               "export <index>...",
```

Comment:
Instead of the default `[--flags]` that cobra is putting at the end, I am currently trying to add meaningful `Use` section, following [cobra](https://pkg.go.dev/github.com/spf13/cobra?utm_source=godoc#Command) source code guidlines, or the GitHub CLI ones: https://primer.style/cli/components/syntax :

```suggestion
		Use:               "export <index> [--scope <scope>...] [--directory]",
```

WDYT?

Reply:
Make sense, would love to start creating guidelines (and documentation?) including design system for the CLI!

Reply:
Changed the import `Use` section as well